acme.sh

```diff
@@ -265,6 +265,31 @@ _usage() {
   printf "\n" >&2
 }
 
+__debug_bash_helper() {
+  # Return extra debug info when running with bash, otherwise return empty
+  # string.
+  local bash_debug=""
```

Comment:
don't use `local` keyword,  it's bash only.

Reply:
Thanks for the feedback. Fixed it to not use `local` until have determined that it is a bash shell.